client/src/components/applications/profil/components/DetailCadastralTable.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react'
+import DetailKatasterLV from './DetailCadastralLV'
+import {Table} from 'reactstrap'
+
+class DetailKatasterTable extends Component {
+  shouldComponentUpdate(nextProps, nextState) {
+    return this.props.kataster !== nextProps.kataster
+  }
+
+  onParcelShow(lv, event) {
```

Comment:
Use arrow function instead of bind. But the best thing here would be to create a presentional component here.